src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,58 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_LENGTH = 5;
```

Comment:
```suggestion
    private static final int ARRAY_LENGTH = 10;
```
please follow task description